packages/@aws-cdk/aws-ec2-alpha/test/vpc-add-method.test.ts

```diff
@@ -398,4 +398,21 @@ describe('Vpc V2 with full control', () => {
     }).toThrow('The VPN Gateway has already been enabled.');
   });
 
+  test('createAcceptorVpcRole creates a restricted role', () => {
```

Comment:
one more test for vpc.createPeeringConnection

Reply:
done! [49e2e4f](https://github.com/aws/aws-cdk/pull/31645/commits/49e2e4f353b0a2847d5ca12f259167fa2894d908)